.pre-commit-config.yaml

```
@@ -0,0 +1,49 @@
+    rev: v1.8.0
+    hooks:
+      - id: mypy
+        additional_dependencies: [types-all]
+        args: ["--config-file", "pyproject.toml"]
+
+  - repo: https://github.com/PyCQA/bandit
+    rev: 1.7.6
+    hooks:
+      - id: bandit
+        args: ["-c", ".bandit"]
+        exclude: tests/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)